docker/suricata/dist/suricata.yaml

```diff
@@ -699,22 +698,30 @@ pcap-file:
   # Warning: 'checksum-validation' must be set to yes to have checksum tested
   checksum-checks: auto
 
-# See "Advanced Capture Options" below for more options, including NETMAP
+# See "Advanced Capture Options" below for more options, including Netmap
 # and PF_RING.
 
 
 ##
-## Step 5: App Layer Protocol Configuration
+## Step 4: App Layer Protocol configuration
 ##
 
-# Configure the app-layer parsers. The protocols section details each
+# Configure the app-layer parsers. The protocol's section details each
 # protocol.
 #
 # The option "enabled" takes 3 values - "yes", "no", "detection-only".
 # "yes" enables both detection and the parser, "no" disables both, and
 # "detection-only" enables protocol detection only (parser disabled).
 app-layer:
   protocols:
+    rfb:
+      enabled: yes
+      detection-ports:
+        dp: 5900, 5901, 5902, 5903, 5904, 5905, 5906, 5907, 5908, 5909
+    # MQTT, disabled by default.
+    mqtt:
+      # enabled: no
+      # max-msg-length: 1mb
```

Comment:
Do you want to enable MQTT?

Reply:
Yes, makes sense since Dionaea has MQTT enabled.